h2o-r/h2o-package/R/models.R

```diff
@@ -259,13 +262,23 @@ h2o.getFutureModel <- function(object,verbose=FALSE) {
   if (scalar) { # scalar == TRUE
     if (type == "H2OModel")
         type <-  "character"
-    if (!inherits(paramValue, type)) {
-      e <- paste0("\"", name , "\" must be of type ", type, ", but got ", class(paramValue), ".\n")
-    } else if ((length(paramDef$values) > 1L) && !(paramValue %in% paramDef$values)) {
-      e <- paste0("\"", name,"\" must be in")
-      for (fact in paramDef$values)
-        e <- paste0(e, " \"", fact, "\",")
-      e <- paste(e, "but got", paramValue)
+    if (name == "seed") {
+	  if(is.character(paramValue) && suppressWarnings(is.na(bit64::as.integer64((paramValue)))))
```

Comment:
package bit64 is not part of the default R installation, it should certainly be optional for H2O to work